core/src/main/java/com/google/googlejavaformat/java/Main.java

```diff
@@ -151,6 +151,13 @@ private int formatFiles(CommandLineOptions parameters, JavaFormatterOptions opti
         allOk = false;
         continue;
       }
+      if (parameters.isValidationMode()) {
+        if (formatted.equals(inputs.get(result.getKey()))) {
+          continue; // no formatting needed, validate next
+        }
+        errWriter.print(result.getKey() + ": needs formatting");
+        return 1; // found first file that needs formatting - exit with error code 1
```

Comment:
can this print all files that need formatting? It's much more useful to get a list of all files that need formatting.